bucket/vieb.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "4.0.0",
+    "description": "Vieb is the Vim Inspired Electron Browser",
+    "homepage": "https://vieb.dev/",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/Jelmerro/Vieb/releases/download/4.0.0/Vieb.4.0.0.exe",
+    "hash": "d5f46c3fb5a8a379866f4a97cd2ad5622fc6430465623e291aefb0dd1f8e6e4a",
+    "shortcuts": [
+        [
+            "Vieb.4.0.0.exe",
```

Comment:
Having autoupdate and version in shortcut does not make sense and will lead to failed installation after autoupdate

Reply:
This is self execute binary. I'll talk with developer if we can have file without version in name.

Reply:
<url>#/view.exe

Reply:
Would it be ok if the zip is versioned, but the exe inside it isn't? For example `Vieb-4.1.0-win.zip` containing `Vieb.exe` inside at the root of the zip?

Reply:
Does not make sense to waste time with different packaging. As already said. Just use URL rewrite `<url>#/view.exe`

Reply:
I have tested both the rewrite option and with zip file. Rewrite was waring however when using Vieb from single exe I had issues with creating shortcuts of my own for example on task bar. I did not experience those issues with zip file. Sine it's already created I used that one.